_posts/2017-06-27-ardublocklysenseBox.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Ardublockly - eine einfache Programmieroberfläche für die senseBox:edu"
+date: 2017-06-27
+author: Mario
+abstract: "Mit Ardublockly wird der Einstieg in die Programmierung der senseBox:edu noch einfacher"
+thumbnail: /images/blog_images/ardublockly_thumbnail.jpg
+image1: /images/blog_images/ardublockly1.gif
+image2: /images/blog_images/ardublockly2.jpg
+lang: de
+---
+Ardublockly für die senseBox:edu
+============
+Mit der senseBox:edu wollen wir einen einfachen Einstieg in die Programmierung und den Bau von Umweltmessgeräten schaffen. Bisher musste immer direkt in der Arduino IDE mithilfe von Programmierbefehlen programmiert werden. Nun ermöglicht eine grafische Oberfläche mit Blöcken einen noch einfacheren und schnelleren Einstieg.
+
+Aktueller Stand und Funktionen
+============
+In die Oberfläche wurden Blöcke für die einzelnen Sensoren der senseBox:edu integriert. Die Sensoren können so einfach ausgelesen werden, wie es im Beispiel gezeigt wird.
+{% include image.html image=page.image1 %}
+Da der Arduino kein Display besitzt müssen anderen Ausgabemöglichkeiten genutzt werden. Hier kann der Nutzer auswählen zwischen der Ausgabe auf dem seriellen Monitor (Kommandozeile) der Arduino IDE, dem speichern auf SD-Karte oder dem direkten übertragen der Messwerte auf die openSenseMap (hierfür muss eine Station vorher registriert werden und eine Wlan oder Lan verbindung vorhanden sein). Die senseBox Blöcke sind in drei Kategorien eingeteilt, den Grundlagen (mit Blöcken für LED, Piepser und der RGB LED), den Sensoren (mit Blöcken für alle Sensoren die in der senseBox:edu enthalten sind) und den verschiedenen Ausgabemöglichkeiten (SD Karte, Kommandozeile oder übertragen an die openSenseMap).
```

Comment:
Ausgabemöglichkeiten & "drei Kategorien" als Liste wäre etwas lesbarer und einfacher zu überfliegen (sind ziemlich lange Sätze :P)

Hier kann der Nutzer auswählen zwischen 
- der Ausgabe auf dem seriellen Monitor (Kommandozeile) der Arduino IDE,
- dem speichern auf SD-Karte 
- oder dem direkten übertragen der Messwerte auf die openSenseMap (hierfür muss eine Station vorher registriert werden und eine WLAN- oder LAN-Verbindung vorhanden sein).

Die senseBox Blöcke sind in drei Kategorien eingeteilt:
- den Grundlagen (mit Blöcken für LED, Piepser und der RGB LED), 
- den Sensoren (mit Blöcken für alle Sensoren die in der senseBox:edu enthalten sind) 
- und den verschiedenen Ausgabemöglichkeiten (SD Karte, Kommandozeile oder übertragen an die openSenseMap).